test/TestDidDocuments.ts

```diff
@@ -0,0 +1,92 @@
+import * as dotenv from 'dotenv';
+import { Keys } from '@ew-did-registry/keys'
+import {IDIDDocument} from "@ew-did-registry/did-resolver-interface"
+
+dotenv.config();
+
+const firstKeys = new Keys({
+  privateKey: process.env.PRIVATE_KEY
+});
+
+const secondKeys = new Keys({
+  privateKey: process.env.PRIVATE_KEY_2
+});
```

Comment:
Similar to the above, I think it is easier just to generate keys. Then we don't need to have environment variables in the automated tests and developers don't need to setup environment variables.
```suggestion
import { Keys } from '@ew-did-registry/keys'
import {IDIDDocument} from "@ew-did-registry/did-resolver-interface"

const firstKeys = new Keys();

const secondKeys = new Keys();
```